examples/src/main/kotlin/examples/inference/saveload/SaveLoadExample.kt

```diff
@@ -0,0 +1,46 @@
+package examples.inference.saveload
+
+import org.jetbrains.kotlinx.dl.api.core.SavingFormat
+import org.jetbrains.kotlinx.dl.api.core.Sequential
+import org.jetbrains.kotlinx.dl.api.core.WritingMode
+import org.jetbrains.kotlinx.dl.api.core.callback.Callback
+import org.jetbrains.kotlinx.dl.api.core.history.BatchTrainingEvent
+import org.jetbrains.kotlinx.dl.api.core.history.TrainingHistory
+import org.jetbrains.kotlinx.dl.api.core.loss.Losses
+import org.jetbrains.kotlinx.dl.api.core.metric.Metrics
+import org.jetbrains.kotlinx.dl.api.core.optimizer.SGD
+import org.jetbrains.kotlinx.dl.dataset.Dataset
+import java.io.File
+
+object SaveLoadExample {
```

Comment:
Could you please add some KDocs here with the high-level description of this object (is it a helper object?)

Reply:
@kokorins SaveLoadExample should be imo a class as we see it can be parametrized